browser-tests/imposter/config/api-config.yaml

```diff
@@ -213,3 +218,107 @@ resources:
         {
           "page": "prove-identity-another-type-photo-id"
         }
+
+  # Journeys where there is an existing identity
```

Comment:
I think these two `/journey/next` routes should go up around line 53 so they are together with the default version.